lib/patchelf/exceptions.rb

```diff
@@ -6,4 +6,6 @@
 module PatchELF
   # Raised on an error during ELF modification.
   class PatchError < ELFTools::ELFError; end
+  class MissingTagError < PatchError; end
```

Comment:
Add descriptions about when will these errors be raised.

e.g. `# Raised when the desired dynamic tag is not found`

Reply:
good enough ?